docs/DeveloperGuide.md

```diff
@@ -206,7 +206,37 @@ Deletion of `Person` from NetConnect is facilitated by `Model#getPersonById(Id)`
   * Pros: Simple implementation of `DeleteCommand#execute(...)`.
   * Cons:
     1. Presence checks required in `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)`.
-    1. Much boilerplate code since `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)` are very similar.
+    1. More boilerplate code since `Model#deletePersonById(Id)` and `Model#deletePersonByName(Name)` are very similar.
+
+### Findnum feature
+
+#### Expected behaviour
+
+The `findnum` command allows users to identify one or more `Person`s from NetConnect using one or more `Phone` numbers. Exceptional behaviour:
+* If any of the phone numbers provided is invalid, an error message is shown.
+* If there are no `Person`s with the given `Phone`, the display is updated to show an empty list.
+
+<puml src="diagrams/FindNumActivityDiagram.puml" alt="FindNumActivityDiagram" />
+
+#### Current implementation
+Given a command `find 98765432`, the `NetConnectParser` recognises the `find` command and first instantiates a `FindNumCommandParser` object. It then passes the command string, where each 'number' is validated using the 
+`Phone#isValidPhone(...)` method. Following which, `FindNumCommandParser` creates a predicate called `PhoneContainsNumbersPredicate`, which is then used to create a `Command` object. 
+
+This `Command` object is a model, and is then passed into the `FindNumCommand` class using the `execute` method. The `FindNumCommand` class then calls the updateFilteredPersonList method, using the predicate created. Now, the `Model` class will update the filtered list of persons to only include persons with the given phone number.
```

Comment:
Sorry what does the "is a model" part mean?

Reply:
Maybe use `Model#updateFilteredPersonList(...)`?

On a separate note, I need to update this in the future since the method has been changed in #84.